src/main/java/nl/tudelft/contextproject/model/level/MazeTile.java

```diff
@@ -90,12 +101,40 @@ public PhysicsControl getPhysicsObject() {
 		if (spatial == null) {
 			this.getSpatial();
 		}
+		if (rigidBody != null) {
+			return rigidBody;
+		}
 
 		CollisionShape sceneShape = CollisionShapeFactory.createMeshShape(spatial);
-		RigidBodyControl rigidBody = new RigidBodyControl(sceneShape, 0);
+		rigidBody = new RigidBodyControl(sceneShape, 0);
 		rigidBody.setPhysicsLocation(spatial.getLocalTranslation());
 		return rigidBody;
 	}
+	
+	/**
+	 * Replace MazeTile.
+	 * This method is used during room creations. As rooms have a MazeTile[][] array before they are placed, this would 
+	 * result in the wrong actual location of the object. Thus the MazeTile has to be replaced as soon as it actually placed
+	 * in the maze.
```

Comment:
please add a newline after this line, so the @ lines are seperated from the method description